tests/playwright/sanity/editor-v2.test.js

```diff
@@ -1,43 +1,41 @@
 const { test, expect } = require( '@playwright/test' );
 const WpAdminPage = require( '../pages/wp-admin-page.js' );
 
-// Skipped until building packages as part of the root build process.
-test.describe.skip( 'Editor v2', () => {
-	const updateExperiment = async ( { browser, testInfo }, experimentState ) => {
-		const context = await browser.newContext();
-		const page = await context.newPage();
+test.describe( 'Editor v2', () => {
+	let editor;
+	let wpAdminPage;
+	let context;
 
-		const wpAdminPage = new WpAdminPage( page, testInfo );
+	test.beforeAll( async ( { browser }, testInfo ) => {
+		context = await browser.newContext();
 
-		await wpAdminPage.setExperiments( { editor_v2: experimentState } );
+		const page = await context.newPage();
 
-		await page.close();
-		await context.close();
-	};
+		wpAdminPage = new WpAdminPage( page, testInfo );
 
-	test.beforeAll( async ( { browser }, testInfo ) => {
-		// Make sure the experiment of editor v2 is enabled during all the tests.
-		await updateExperiment( { browser, testInfo }, true );
+		await wpAdminPage.setExperiments( { editor_v2: true } );
+
+		editor = await wpAdminPage.openNewPage();
 	} );
 
-	test.afterAll( async ( { browser }, testInfo ) => {
-		// Disable the experiment of editor v2 after all the tests.
-		await updateExperiment( { browser, testInfo }, false );
+	test.afterAll( async () => {
+		await wpAdminPage.setExperiments( { editor_v2: false } );
 	} );
 
-	test( 'validate the top bar appears', async ( { page }, testInfo ) => {
-		// Arrange.
-		const wpAdmin = new WpAdminPage( page, testInfo );
+	test( 'check that app-bar exists', async () => {
+		// Act
+		const wrapper = await editor.page.locator( '#elementor-editor-wrapper-v2' );
 
-		// Act.
-		const editor = await wpAdmin.useElementorCleanPost();
+		await wrapper.getByRole( 'button', { name: 'Post Settings' } ).click();
 
-		// Assert
-		const wrapper = await editor.page.locator( '#elementor-editor-v2-top-bar' );
+		await editor.page.getByLabel( 'Title', { exact: true } ).fill( 'Test page' );
 
+		await wrapper.getByRole( 'button', { name: 'Test page' } ).waitFor();
+
+		// Assert
```

Comment:
How about something like (notice that this API requires PNG screenshots):
```javascript
await expect( wrapper ).toHaveScreenshot( 'editor-v2-wrapper.png' );
```

Reply:
Changed